server/services/hearingsService.ts

```diff
@@ -49,18 +50,24 @@ export default class HearingsService {
       reason: adjournReason,
       plea,
     }
-    return new ManageAdjudicationsClient(user).createAdjourn(chargeNumber, hearingOutcomeDetails)
+    return new ManageAdjudicationsUserTokensClient(user).createAdjourn(chargeNumber, hearingOutcomeDetails)
   }
 
   async getHearingOutcome(chargeNumber: string, user: User) {
-    const adjudication = await new ManageAdjudicationsClient(user).getReportedAdjudication(chargeNumber)
+    const token = await this.hmppsAuthClient.getSystemClientToken(user.username)
```

Comment:
To construct this `token` we make call to HMPPS Auth and pass the [clientToken](https://github.com/ministryofjustice/hmpps-manage-adjudications/blob/9a701b305c0576870bcaefc485942b7760c8a33c/server/data/hmppsAuthClient.ts#L13-L16) which corresponds to the `manage-adjudications-client` client credentials